### PR TITLE
Finished jsx refactor

### PR DESCRIPTION
**Pull request by e5chang**

### CODE REVIEW

**Comment by waynarbocangel** [comment]
I can't tell if this contains the settings panels. If it doesn't it's fine I can work around those in a future PR.

File: codebase/back-end/security/.env [hunk 1]
```diff
@@ -1 +0,0 @@
-DB=mongodb+srv://z7GK4UuS26KryKaX:maxJQoyjzMJCPGlH@cse110-team11.g6uiw.mongodb.net/cse110-team11
```
Comment (ehsonp) [inline codebase/back-end/security/.env:1]: Was this supposed to be there? @e5chang

Comment (ehsonp) [reply codebase/back-end/security/.env:1]: nevermind thought it was in a different folder

File: src/front-end/components/audioBlock.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+/* eslint-disable */
+/** @jsx createElement */
+/** @jsxFrag createFragment */
+import { createElement, createFragment } from "../jsxEngine.js";
+/* eslint-enable */
+
+let template = <template>
+	<div id="editorIcons" class="paragraphIcons"><img src="../public/resources/plusIcon.png" class="unfocusedIcons" id="plus" /><img src="../public/resources/sixDotIcon.png" class="unfocusedIcons" id="more" /></div>
+	<form action='/api/audio' method="post" enctype="multipart/form-data">
+  		<input type='file' name="audio" />
```
Comment (ehsonp) [inline src/front-end/components/audioBlock.jsx:19]: I dont see where you import the audioBlock.css file

**Review by ehsonp — COMMENTED** [review]
Just wondering why Audio Block jsx does not have a audioblock.css imported

File: src/front-end/components/audioBlock.jsx (hunk 2 above)
Comment (waynarbocangel) [reply src/front-end/components/audioBlock.jsx:19]: It's actually not imported anywhere. We didn't have time to fully finalize it so it's just stand alone. Same with the ImageBlock.

Comment (waynarbocangel) [reply src/front-end/components/audioBlock.jsx:19]: Oh wait I miss read that. I'll add it and recommit. Thanks!

Comment (ehsonp) [reply src/front-end/components/audioBlock.jsx:19]: cool ill approve right after

Comment (waynarbocangel) [reply src/front-end/components/audioBlock.jsx:19]: Just pushed the changes. Thanks for catching it!

**Review by ehsonp — APPROVED** [review]
good to merge

**Comment by forschoolandnothingelse** [comment]
Why are there 48 commits tied to this PR? I can't figure out what I'm even supposed to be looking at.

**Comment by waynarbocangel** [comment]
> Why are there 48 commits tied to this PR? I can't figure out what I'm even supposed to be looking at.

If you look at the vast majority of them (the first 40 or so) are commits already in main. Remember this is the front-end branch and because we have not been committing here yet the branch itself is behind everything we have done basically. If you wanna look at the changes look at all the files in the components directory in the front-end. The index also has minor changes compared to the version in main. Other than that there are no changes other than references to the changed files in like import statements on other files. Lmk if that clarifies it a bit for you.

**Review by rr-zhou — APPROVED** [review]

**Pull request merged**